raspimouse/src/raspimouse_component.cpp

```diff
@@ -230,6 +232,10 @@ CallbackReturn Raspimouse::on_activate(const rclcpp_lifecycle::State &)
     light_sensors_timer_->reset();
   }
 
+  // Set the motor on/off
+  const auto INIT_MOTOR_POWER = get_parameter(INIT_MOTOR_POWER_PARAM).get_value<bool>();
+  set_motor_power(INIT_MOTOR_POWER);
```

Comment:
INIT_MOTOR_POWERはここでしか使用されていないので、
変数を用意しなくてOKです。
```suggestion
  set_motor_power(get_parameter(INIT_MOTOR_POWER_PARAM).get_value<bool>());
```